index.html

```diff
@@ -61,9 +60,152 @@ <h1>I’m Alexandra.   Glad   to  see      you!</h1>
+                            </ul>
+                        </div>
+                        <p class="">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <div class="lower_part">
+                            <ul class="project-tech-stack">
+                                <li><a>HTML</a></li>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </section>
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectfour.png">
+                            <h3 class="ptitle">Multi-Post Stories</h3>
+                        </div>
+                        <div class="p-details">
+                            <ul>
+                                <li>CANOPY</li>
+                                
+                                <li>Back End Dev</li>
+                               
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="lower_part">
+                        <ul class="project-tech-stack">
+                            <li><a>HTML</a></li>
+                            <li><a>CSS</a></li>
+                            <li><a>Javascript</a></li>
+                        </ul>
+                        <a class="button">See project</a>
+                        </div>
+                    </section>
+            </section>
+            
+            
+            <section class="about-me">
+                <h2 class="titletext">About Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and 
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+                <p class="letconnect"> LET’S CONNECT</p>
+                           
+                       
+
+                    
+                <div class="socialmedia">
+                    <ul class="media">                           
+                        
+                        <li><a class="media"><img src="img/media-3.png"></a></li>
+                        <li><a class="media"><img src="img/media-4.png"></a></li>
+                        <li><a class="media"><img src="img/media-1.png"></a></li>
+                        <li><a class="media"><img src="img/media-5.png"></a></li>
+                        <li><a class="media"><img src="img/media-2.png"></a></li>
+                    </ul>
+
+                </div>
```

Comment:
Great job on adding the section social icon list! However, there are a few adjustments that need to be made to align it with the design:

- [ ] Ensure that proper spacing is added between the social icons. The spacing should be consistent and match the design guidelines. This will create a visually pleasing and organized layout.

- [ ] Pay attention to the alignment of the social icons. Make sure they are properly aligned horizontally and vertically within the section. The consistent alignment will contribute to a polished and professional appearance.

By updating the spacing and alignment of the social icon list, you will bring it in line with the design and improve the overall visual presentation of the webpage. Keep up the good work and strive for design accuracy!

| Your | Professional |
| --- | --- |
|![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/55840999/809080d3-7616-4679-ae99-11416b25d6f6) | ![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/55840999/dab8600e-6023-4dcd-8aae-838ba7817355) |